After upgrade from 3.16.2 to 3.16.4 - Cannot find module '@tailwindcss/container-queries'

After upgrading to the latest version (3.16.2 to 3.16.4), my build process has stopped working due to the missing module '@tailwindcss/container-queries'. Downgrading to 3.16.2 resolves the issue.

Fragment of my local package.json:
```
    "@tailwindcss/container-queries": "^0.1.1",
```

Build excerpt:
```
 DEBUG [82df103b]       #21 5.054 yarn install v1.22.19
 DEBUG [82df103b]       #21 5.174 [1/4] Resolving packages...
 DEBUG [82df103b]       #21 5.257 success Already up-to-date.
 DEBUG [82df103b]       #21 5.261 Done in 0.21s.
 DEBUG [82df103b]       #21 5.483 yarn run v1.22.19
 DEBUG [82df103b]       #21 5.502 $ tailwindcss --postcss --minify -i ./app/assets/stylesheets/application.tailwind.css -o ./app/assets/builds/application.css
 DEBUG [82df103b]       #21 6.530 
 DEBUG [82df103b]       #21 6.531 Rebuilding...
 DEBUG [82df103b]       #21 8.047 
 DEBUG [82df103b]       #21 8.047 Done in 1872ms.
 DEBUG [82df103b]       #21 8.085 Done in 2.60s.
 DEBUG [82df103b]       #21 8.463 yarn install v1.22.19
 DEBUG [82df103b]       #21 8.498 [1/4] Resolving packages...
 DEBUG [82df103b]       #21 8.552 success Already up-to-date.
 DEBUG [82df103b]       #21 8.558 Done in 0.10s.
 DEBUG [82df103b]       #21 8.786 yarn run v1.22.19
 DEBUG [82df103b]       #21 8.803 $ node esbuild.config.mjs
 DEBUG [82df103b]       #21 9.057 Done in 0.27s.
 DEBUG [82df103b]       #21 9.074 [Avo->] Adding yarn dependencies
 DEBUG [82df103b]       #21 9.370 (node:132) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
 DEBUG [82df103b]       #21 9.370 (Use `node --trace-deprecation ...` to show where the warning was created)
 DEBUG [82df103b]       #21 11.11 [Avo->] Linking avo-advanced to /usr/local/bundle/ruby/3.3.0/gems/avo-advanced-3.16.4
 DEBUG [82df103b]       #21 11.12 [Avo->] Linking avo-pro to /usr/local/bundle/ruby/3.3.0/gems/avo-pro-3.16.4
 DEBUG [82df103b]       #21 11.12 [Avo->] Linking avo-dynamic_filters to /usr/local/bundle/ruby/3.3.0/gems/avo-dynamic_filters-3.16.4
 DEBUG [82df103b]       #21 11.13 [Avo->] Linking avo-dashboards to /usr/local/bundle/ruby/3.3.0/gems/avo-dashboards-3.16.4
 DEBUG [82df103b]       #21 11.14 [Avo->] Linking avo-menu to /usr/local/bundle/ruby/3.3.0/gems/avo-menu-3.16.4
 DEBUG [82df103b]       #21 11.16 [Avo->] Linking avo.base.css to /usr/local/bundle/ruby/3.3.0/gems/avo-3.16.4/app/assets/builds/avo.base.css
 DEBUG [82df103b]       #21 11.17 [Avo->] Linking tailwind.preset.js to /usr/local/bundle/ruby/3.3.0/gems/avo-3.16.4/tailwind.preset.js
 DEBUG [82df103b]       #21 12.06 
 DEBUG [82df103b]       #21 12.07 Rebuilding...
 DEBUG [82df103b]       #21 12.94 Error: Cannot find module '@tailwindcss/container-queries'
 DEBUG [82df103b]       #21 12.94 Require stack:
 DEBUG [82df103b]       #21 12.94 - /usr/local/bundle/ruby/3.3.0/gems/avo-3.16.4/tailwind.preset.js
 DEBUG [82df103b]       #21 12.94     at Module._resolveFilename (node:internal/modules/cjs/loader:1219:15)
 DEBUG [82df103b]       #21 12.94     at Function.resolve (node:internal/modules/helpers:133:19)
 DEBUG [82df103b]       #21 12.94     at _resolve (/rails/node_modules/jiti/dist/jiti.js:1:246378)
 DEBUG [82df103b]       #21 12.94     at jiti (/rails/node_modules/jiti/dist/jiti.js:1:249092)
 DEBUG [82df103b]       #21 12.94     at /usr/local/bundle/ruby/3.3.0/gems/avo-3.16.4/tailwind.preset.js:149:5
 DEBUG [82df103b]       #21 12.94     at evalModule (/rails/node_modules/jiti/dist/jiti.js:1:251913)
 DEBUG [82df103b]       #21 12.94     at jiti (/rails/node_modules/jiti/dist/jiti.js:1:249841)
 DEBUG [82df103b]       #21 12.94     at /rails/config/avo/tailwind.config.js:1:94
 DEBUG [82df103b]       #21 12.94     at evalModule (/rails/node_modules/jiti/dist/jiti.js:1:251913)
 DEBUG [82df103b]       #21 12.94     at jiti (/rails/node_modules/jiti/dist/jiti.js:1:249841) {
 DEBUG [82df103b]       #21 12.94   code: 'MODULE_NOT_FOUND',
 DEBUG [82df103b]       #21 12.94   requireStack: [ '/usr/local/bundle/ruby/3.3.0/gems/avo-3.16.4/tailwind.preset.js' ]
 DEBUG [82df103b]       #21 12.94 }
 DEBUG [82df103b]       #21 12.96 error Command failed with exit code 1.
```

--- a/lib/tasks/avo_tasks.rake
+++ b/lib/tasks/avo_tasks.rake
@@ -134,5 +134,5 @@ desc "Installs yarn dependencies for Avo"
 task "avo:yarn_install" do
   # tailwind.preset.js needs this dependencies in order to be required
   puts "[Avo->] Adding yarn dependencies"
-  `yarn add tailwindcss @tailwindcss/forms @tailwindcss/typography --cwd #{Avo::Engine.root}`
+  `yarn add tailwindcss @tailwindcss/forms @tailwindcss/typography @tailwindcss/container-queries --cwd #{Avo::Engine.root}`
 end

